source/draw/RasterMonochrome.ooc

```diff
@@ -25,7 +25,7 @@ import Image, FloatImage
 import Color
 import Canvas, RasterCanvas
 
-MonochromeRasterCanvas: class extends RasterCanvas {
+MonochromeRasterCanvas: class extends PackedRasterCanvas {
```

Comment:
We should set a naming standard here (`MonochromeRasterCanvas` was my fault I think).
`RasterImage` -> `RasterPacked` -> `RasterMonochrome`
`RasterCanvas` -> `RasterCanvasPacked` -> `RasterCanvasMonochrome`

What do you propose?